oidc_cli/client/client.go

```diff
@@ -22,6 +22,7 @@ type Client struct {
 	oauthConfig *oauth2.Config
 	verifier    *oidc.IDTokenVerifier
 	server      *server
+	customMsgs  map[string]string
```

Comment:
```suggestion
	customMessages  map[OIDCStatus]string
```
- I think we should make a type
 ```
type oidcStatus string
var oidcStatusSuccess oidcStatus = "success"

type clientOpts func(*Client) *Client

func SetSuccessMessage func(successMessage string) clientOpts {
   return func(c *Client) *Client {
      c.customMessages[oidcStatusSuccess] = successMessage
      return c
   }
}
```
And then use that so we can enumerate the success status here